src/components/shared/form_fields/TextField.vue

```diff
@@ -56,4 +58,8 @@ const onUpdateModel = ( newValue: string|number ): void => {
 
 <style scoped lang="scss">
```

Comment:
You can remove `scoped`. If you see that elsewhere please also delete it, PhpStorm auto-adds it when you create a new Vue single component file and it's sneaked in in a few places.